src/agent0/chainsync/dashboard/usernames.py

```diff
@@ -7,16 +7,13 @@
 import pandas as pd
 
 
-def build_user_mapping(
-    addresses: pd.Series, addr_to_username: pd.DataFrame, username_to_user: pd.DataFrame
-) -> pd.DataFrame:
+def build_user_mapping(addresses: pd.Series, addr_to_username: pd.DataFrame) -> pd.DataFrame:
     """Given a pd.Series of wallet addresses, we build a corresponding dataframe that contains
```

Comment:
Can you add a top-level summary next to `"""` and move this description down w/ a newline separating?